test/planet_test.rb

```diff
@@ -7,4 +7,10 @@ def test_it_exists
     planet = Planet.new
     assert_instance_of Planet, planet
   end
-end 
+
+  def test_it_can_spin
+    planet = Planet.new
+
+    assert planet.spin?
```

Comment:
I appreciate the use of assert, do we want to consider the false return value?

Reply:
Oooh yes! good idea